desktop/src/renderer/src/components/FileUploadArea.tsx

```diff
@@ -104,6 +106,12 @@ export const FileUploadArea = ({
       if (isAccepted) {
         onDrop(droppedFile);
       } else {
+        toast.error(Toast, {
+          data: {
+            title: "Wrong file type",
+            message: `File type not accepted. Please upload an ${fileExtensions} file.`,
```

Comment:
Please note if file extensions .xlsx and .json - might worth fix the Browse file button display too
- Browser file (.xlsx, .json) - with one space in between according to the designer
- Toast popup - should be something like "upload an .xmls or .json file."

![image](https://github.com/user-attachments/assets/eec6202b-a74c-4fce-9586-ce085fe597fc)